test/plugin/test_out_google_cloud.rb

```diff
@@ -316,4 +316,25 @@ def get_number(field)
   def null_value
     nil
   end
+
+  # 'responseSize' and 'requestSize' are Integers in the gRPC proto, yet Strings
+  # in REST API client.
+  # TODO(qingling128): Address this accordingly once the following  question is
```

Comment:
Uber-nit: two spaces between "following" and "question".

Reply:
Fixed.